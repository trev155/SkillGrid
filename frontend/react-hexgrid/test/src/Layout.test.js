import React from 'react';
import renderer from 'react-test-renderer';

import Layout from '../../src/Layout';

test('Layout should render correctly with default props', () => {
  const tree = renderer.create(
    <Layout
      className={'test1'}
    >
      <div>child</div>
    </Layout>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Layout should render correctly with custom props', () => {
  const tree = renderer.create(
    <Layout
      className={'test2'}
      flat={false}
      origin={{ x: 2, y: 4 }}
      size={{ x: 12, y: 14 }}
      spacing={2.0}
    >
      <div>child</div>
    </Layout>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
