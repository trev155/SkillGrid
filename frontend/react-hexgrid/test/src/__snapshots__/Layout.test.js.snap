// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render correctly with custom props 1`] = `
<g
  className="test2"
>
  <div>
    child
  </div>
</g>
`;

exports[`Layout should render correctly with default props 1`] = `
<g
  className="test1"
>
  <div>
    child
  </div>
</g>
`;
